docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+     "key-uid" "a5424fb033f5cc66dce9cbbe464426b6feff70ca40aa952c56247aaeaf4764a9", 
+     "wallet-public-key" "04d6fab73772933215872c239787b2281f3b10907d099d04b88c861e713bd2b95883e0b1710a266830da29e76bbf6b87ed034ab139e36cc235a1b2a5b5ddfd4e91", 
+     "public-key" "0437eef85e669f800570f444e64baa2d0580e61cf60c0e9236b4108455ec1943f385043f759fcb5bd8348e32d6d6550a844cf24e57f68e9397a0f7c824a8caee2d", 
+     "instance-uid" "2268254e3ed7898839abe0b40e1b4200"}
+    ```
+3. `status-im.hardwallet.card/get-application-info`
+    params: 
+    ```clojure
+    {:pairing "ACEWbvUlordYWOE6M1Narn/AXICRltjyuKIAn4kkPXQG"}
+    ```
+    response:
+    ```clojure
+    {"paired?" true, 
+     "has-master-key?" true, 
+     "app-version" "2.2", 
+     "free-pairing-slots" 4, 
+     "pin-retry-counter" 3, 
+     "puk-retry-counter" 5, 
+     "initialized?" true, 
+     "secure-channel-pub-key" "04b079ac513d5e0ebbe9becbae1618503419f5cb59edddc7d7bb09ce0db069a8e6dec1fb40c6b8e5454f7e1fcd0bb4a0b9750256afb4e4390e169109f3ea3ba91d", 
+     "key-uid" "a5424fb033f5cc66dce9cbbe464426b6feff70ca40aa952c56247aaeaf4764a9", 
+     "instance-uid" "2268254e3ed7898839abe0b40e1b4200"}
+    ```
+
+## Tx signing
+1. `status-im.hardwallet.card/get-application-info`
+   params: 
+   ```clojure
+   {:pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h", 
+    :on-success :hardwallet/sign}
+   ```
+   response:
+   ```clojure
+    {"paired?" true, 
+     "has-master-key?" true, 
+     "app-version" "2.2", 
+     "free-pairing-slots" 4, 
+     "pin-retry-counter" 3, 
+     "puk-retry-counter" 5, 
+     "initialized?" true, 
+     "secure-channel-pub-key" "0476d11f2ccdad4e7779b95a1ce063d7280cb6c09afe2c0e48ca0c64ab9cf2b3c901d12029d6c266bfbe227c73a802561302b2330ac07a3270fc638ad258fced4a", 
+     "key-uid" "d5c8cde8085e7a3fcf95aafbcbd7b3cfe32f61b85c2a8f662f60e76bdc100718", 
+     "instance-uid" "e20e27bfee115b431e6e81b8e9dcf04c"}
+    ```
+
+2. `status-im.hardwallet.card/sign` 
+   params:
+   ```clojure
+   {:hash "92fc7ef54c3e0c42de256b93fbf2c49dc6948ee089406e204dec943b7a0142a9", 
+    :pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h", 
+    :pin "123123", 
+    :path "m/44'/60'/0'/0/0"}
+   ```
+   response: `5d2ca075593cf50aa34007a0a1df7df14a369534450fce4a2ae8d023a9d9c0e216b5e5e3f64f81bee91613318d01601573fdb15c11887a3b8371e3291e894de600`
+
+## Account derivation
+1. `status-im.hardwallet.card/verify-pin` 
+    params:
+    ```clojure
+    {:pin "123123", 
+     :pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h"}
+    ```
+    response: `3`
+1. `status-im.hardwallet.card/export-key`
+    params:
+    ```clojure
+    {:pin "123123", 
+     :pairing "ALecvegKyOW4szknl01yYWx60GLDK5gDhxMgJECRZ+7h", 
+     :path "m/44'/60'/0'/0/1"}
+    ```
+    response: `046d1bcd2310a5e0094bc515b0ec995a8cb59e23d564094443af10011b6c00bdde44d160cdd32b4b6341ddd7dc83a4f31fdf60ec2276455649ccd7a22fa4ea01d8` (account's `public-key`)
+    
+## Where are the keys stored?
```

Comment:
How does this interact with https://specs.status.im/spec/2#security-considerations ? Might be worth cross linking / adding detail in that spec too

Reply:
@oskarth do you ask me or @corpetty ?

Reply:
you, it should probably be linked

Reply:
it is empty at the moment, do you want me just to add that link to the doc?

Reply:
I mean security considerations should be covered when it comes to key storage. Whether it should be part of this doc or the other doc is up to you guys. Whichever way it should probably be cross linked as it seems related.

If this too big for PR we can pull it out into a separate issue.

@rajeevgopalakrishna can probably assist with how to phrase these considerations

Reply:
I can add these specifics in a separate PR.

Reply:
> If this too big for PR we can pull it out into a separate issue
yep pr is big already